kano_login/first_screen.py

```diff
@@ -66,6 +66,12 @@ def __init__(self):
 
 
 class FirstScreen(FirstScreenTemplate):
+
+    webengine = '/usr/bin/kano-webengine'
+    web_api = 'http://10.49.104.215:8080/'
```

Comment:
hardcoded ip and port... mmmm... could we live with a command line option, environment or something?

Reply:
Methinks that we should move this to [`kano_world.config`](https://github.com/KanoComputing/kano-profile/blob/master/kano_world/config.py) and that way we can also set it through `/etc/kano-world.conf`